subprojects/core/src/main/java/org/gradle/api/internal/tasks/SnapshotTaskInputsBuildOperationResult.java

```diff
@@ -114,8 +146,25 @@ public String getPropertyName() {
         }
 
         @Override
+        public Set<String> getPropertyAttributes() {
+            InputFilePropertySpec propertySpec = propertySpec(propertyName);
+            return ImmutableSortedSet.of(
+                FilePropertyAttribute.fromNormalizerClass(propertySpec.getNormalizer()).name(),
+                FilePropertyAttribute.from(propertySpec.getDirectorySensitivity()).name(),
+                FilePropertyAttribute.from(propertySpec.getLineEndingNormalization()).name()
+            );
+        }
+
+        @Override
+        @SuppressWarnings({"deprecation", "RedundantSuppression"})
```

Comment:
This annotation can also go away.

Reply:
Given the gradle build uses `-Werror`, removing this fails.
I replaced it by `@Deprecated`, looking at other similar cases in the codebase